client/src/components/locations.jsx

```diff
@@ -69,21 +86,21 @@ class Locations extends React.Component {
 
   render() {
     return (
-      <div>
-        <div className='container'>
-          <SliderButton onClick={this.slideLeft}>
+      <SliderContainer >
+        <SliderCardContainer>
+          {/* <SliderButton onClick={this.slideLeft}>
```

Comment:
I think i have also done this in my PRs but i doubt having commented out code in master is something you want

Reply:
@danielharmon I need it eventually. It was on the locations tab then it got moved to materials. I haven't made the call to the db. so when that happens, these commented out codes will be needed